app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: float,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        price = (
+            car.comfort_class
+            * (self.clean_power - car.clean_mark)
+            * self.average_rating
+            / self.distance_from_city_center
+        )
+        return round(price, 1)
+
+    def wash_single_car(self, car: Car) -> float:
+        return self.calculate_washing_price(car)
+
+    def serve_cars(self, cars: list) -> float:
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.wash_single_car(car)
+                car.clean_mark = self.clean_power
+        return round(income, 1)
+
+    def rate_service(self, mark: int) -> None:
+        if 0 <= mark <= 5:
+            all_marks_sum = self.average_rating * self.count_of_ratings + mark
+            self.count_of_ratings += 1
+            self.average_rating =\
+                round(all_marks_sum / self.count_of_ratings, 1)
```

Comment:
Why is `\` used here? It looks like it's not needed, as it provides poor readability.